<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>svHelloData</class>
 <widget class="QWidget" name="svHelloData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="helloLabel">
        <property name="text">
         <string>Enter Class Here: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="helloLineEdit">
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string>Hello World</string>
        </property>
        <property name="placeholderText">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="helloPushButton">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="helloPushType">
     <property name="text">
      <string>Push to Search For Class</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
